apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2068,6 +2456,21 @@ static int fs_sample_launcher(int argc, char **args)
 	libc_stdio_setbuf_tc();
 	libc_stdio_setvbuf_tc();
 #endif
+	libc_stdio_meminstream_tc();
+	libc_stdio_memoutstream_tc();
+	libc_stdio_memsistream_tc();
+	libc_stdio_memsostream_tc();
+	libc_stdio_nullinstream_tc();
+	libc_stdio_nulloutstream_tc();
+	libc_stdio_rawinstream_tc();
+	libc_stdio_rawoutstream_tc();
+	libc_stdio_rawsistream_tc();
+	libc_stdio_rawsostream_tc();
+	libc_stdio_stdinstream_tc();
+	libc_stdio_stdoutstream_tc();
+	libc_stdio_stdsistream_tc();
+	libc_stdio_stdsostream_tc();
+	libc_stdio_zeroinstream_tc();
```

Comment:
please check the alphabet sequence

Reply:
it seems correct

Reply:
libc_stdio_zeroinstream_tc should be placed after libc_stdio_ungetc_tc

Reply:
ok will do that, sorry did not spot that